core/src/bundle_stage.rs

```diff
@@ -292,16 +344,32 @@ impl BundleStage {
         bank_start: &BankStart,
         bundle_stage_leader_stats: &mut BundleStageLeaderStats,
         max_bundle_retry_duration: &Duration,
+        reserved_space: &mut BundleReservedSpace,
     ) -> BundleStageResult<()> {
         if sanitized_bundle.transactions.is_empty() {
             return Ok(());
         }
 
         let tx_costs = qos_service.compute_transaction_costs(sanitized_bundle.transactions.iter());
+        let bundle_cost: u64 = tx_costs.iter().map(|c| c.sum()).sum();
+        let cost_tracker = &mut bank_start.working_bank.write_cost_tracker().unwrap();
+        // Increase blcok cost limit for bundles
+        cost_tracker.set_limits(
```

Comment:
im a little lost on this stuff, can you explain more?

Reply:
are you essentially increasing block limit, adding these txs, then decreasing it again to fit in?